src/components/Caculator.js

```diff
@@ -1,32 +1,92 @@
-const MyInput = () => <input type="text" className="inputfield" value={0} />;
-const MyBtn = () => (
-  <div className="calc-btn-grid">
-    <button type="button" className="btn btn-number">AC</button>
-    <button type="button" className="btn btn-number">+/-</button>
-    <button type="button" className="btn btn-number">%</button>
-    <button type="button" className="btn btn-sign">/</button>
-    <button type="button" className="btn btn-number">7</button>
-    <button type="button" className="btn btn-number">8</button>
-    <button type="button" className="btn btn-number">9</button>
-    <button type="button" className="btn btn-sign">X</button>
-    <button type="button" className="btn btn-number">4</button>
-    <button type="button" className="btn btn-number">5</button>
-    <button type="button" className="btn btn-number">6</button>
-    <button type="button" className="btn btn-sign">-</button>
-    <button type="button" className="btn btn-number">1</button>
-    <button type="button" className="btn btn-number">2</button>
-    <button type="button" className="btn btn-number">3</button>
-    <button type="button" className="btn btn-sign">+</button>
-    <button type="button" className="btn btn-number btn-zero">0</button>
-    <button type="button" className="btn btn-number">.</button>
-    <button type="button" className="btn btn-sign">+</button>
-  </div>
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prop-types
+const MyInput = ({ value }) => (
+  <input type="text" className="inputfield" value={value} readOnly />
 );
```

Comment:
- [ ] Kindly remove this `eslint-disable` statement. Eslint helps JavaScript developers to write clean, maintainable and readable codes. Using the `eslint-disable` statement defeats this purpose. You can read this [article](https://www.freecodecamp.org/news/how-to-use-proptypes-in-react/#:~:text=import%20PropTypes%20from%20'prop%2Dtypes,%2F%3E%20\)%20%7D%3B%20Count.) or this [article](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html) to learn how to use prop-types in a React component. 👍 💻